Formula/corectl.rb

```diff
@@ -33,10 +34,7 @@ def install
 
     cd path do
       system "opam", "init", "--no-setup"
-      qcow_format_revision = build.head? ? "master" : "96db516d97b1c3ef2c7bccdac8fb6cfdcb667a04"
-      system "opam", "pin", "add", "qcow-format",
-        "https://github.com/mirage/ocaml-qcow.git##{qcow_format_revision}"
-      system "opam", "install", "uri", "qcow-format", "ocamlfind"
+      system "opam", "install", "uri", "ocamlfind", "qcow-format"
```

Comment:
consistency. it's part of stock ocam tree now. i'm not tagging uri or ocamlfind either